tests/unit-tests/gconstruct/test_construct_graph.py

```diff
@@ -719,6 +719,31 @@ def check_classification_no_split(res):
     res = process_labels(data, ops)
     check_classification(res)
 
```

Comment:
Can we add a separate test function like check_edge_classification_no_split

Reply:
Better not to separate the check_edge_classification outside, because we need check_classification which is inside the test_label function, but I can move the link prediction one outside